dotcom-rendering/src/web/lib/hiddenStyles.tsx

```diff
@@ -0,0 +1,7 @@
+import { css } from '@emotion/react';
+
+export const hiddenStyles = css`
+	&.hidden {
+		display: none;
+	}
+`;
```

Comment:
Is there a reason hiddenStyles is defined in its own file?

Reply:
This was to follow the same pattern that was used for revealStyles
https://github.com/guardian/dotcom-rendering/blob/f7a9bc828b4959510a24715672ff36805db1d742/dotcom-rendering/src/web/lib/revealStyles.ts